backtrace-library/build.gradle

```diff
@@ -49,7 +49,10 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.google.code.gson:gson:2.8.5'
     implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.squareup:tape:1.2.3'
+    implementation 'org.apache.commons:commons-lang3:3.0'
     testImplementation 'junit:junit:4.12'
+    androidTestImplementation 'org.mockito:mockito-core:1.10.19'
```

Comment:
why we need this lib?

Reply:
I use this lib for `FileUtils.moveFile` in `BacktraceBase::handleExistingBreadcrumbLogFile`. I want to move the previously found stale breadcrumbs log somewhere else so we can keep logging breadcrumbs for the new session. FileUtils.moveFile seems like a much more convenient alternative to doing that myself.

Reply:
Ah I see, we don't need this dependency:

    implementation 'org.jetbrains:annotations-java5:15.0'

I will remove this unnecessary dependency